test/selenium/selenium-kdump.py

```diff
@@ -65,26 +67,38 @@ def testKdumpBasePage(self):
 
     def check_type(self, old_text, new_text, selector_item, textbox_dict):
         self.wait_text("Service is running", cond=visible)
-        self.click(self.wait_text(old_text, cond=visible))
+        self.click(self.wait_text(old_text, cond=clickable))
         self.select_by_text(self.wait_id("kdump-settings-location",
                                          cond=clickable),
                             selector_item)
         for textbox_key in textbox_dict:
             location = self.wait_id(textbox_key)
             self.send_keys(location, textbox_dict[textbox_key])
-        self.click(self.wait_text("Apply", cond=visible))
+        self.click(self.wait_text("Apply", cond=clickable))
+        # extend waiting for regeneration of initrd image, may take long time
+        self.wait_id("kdump-settings-dialog", cond=visible, reversed_cond=True, overridetry=self.default_try * 3)
```

Comment:
Heh, I recently added something similar, but it turned out that it wasn't necessary after all. You can just use `cond=invisible`, right?

Reply:
Yes, I've seen your code and I saw them useful, but you've remoted it then :-) theoretically yes as opposite of visible, but probably would be more clear, to wait to dialog window disapper with this. or in case of another conditions, eg. clickable, there is no opossite invisible, but potentially, ``unclickable`` what may be useful (eg. button is there, but impossible to click because you are not root)